index.js

```diff
@@ -1,27 +1,27 @@
-const baudrates = document.getElementById("baudrates");
-const connectButton = document.getElementById("connectButton");
-const disconnectButton = document.getElementById("disconnectButton");
-const resetButton = document.getElementById("resetButton");
-const consoleStartButton = document.getElementById("consoleStartButton");
-const consoleStopButton = document.getElementById("consoleStopButton");
-const eraseButton = document.getElementById("eraseButton");
-const programButton = document.getElementById("programButton");
-const filesDiv = document.getElementById("files");
-const terminal = document.getElementById("terminal");
-const programDiv = document.getElementById("program");
-const consoleDiv = document.getElementById("console");
-const lblBaudrate = document.getElementById("lblBaudrate");
-const lblConnTo = document.getElementById("lblConnTo");
-const tableBody = document.getElementById("tableBody");
+const baudrates = document.getElementById('baudrates');
+const connectButton = document.getElementById('connectButton');
+const disconnectButton = document.getElementById('disconnectButton');
+const resetButton = document.getElementById('resetButton');
+const consoleStartButton = document.getElementById('consoleStartButton');
+const consoleStopButton = document.getElementById('consoleStopButton');
+const eraseButton = document.getElementById('eraseButton');
+const programButton = document.getElementById('programButton');
+const filesDiv = document.getElementById('files');
+const terminal = document.getElementById('terminal');
+const programDiv = document.getElementById('program');
+const consoleDiv = document.getElementById('console');
+const lblBaudrate = document.getElementById('lblBaudrate');
+const lblConnTo = document.getElementById('lblConnTo');
+const tableBody = document.getElementById('tableBody');
 const table = document.getElementById('fileTable');
 const alertDiv = document.getElementById('alertDiv');
 
-//import { Transport } from './cp210x-webusb.js'
-import { Transport } from './webserial.js'
-import { ESPLoader } from './ESPLoader.js'
-import { ESPError } from './error.js'
+// import { Transport } from './cp210x-webusb.js'
+import * as esptooljs from "./bundle.js";
```

Comment:
I hope in the future we can just import this from unpkg like we suggest for ESP Web Tools:

https://esphome.github.io/esp-web-tools/#add-website

Reply:
I think we can release the bundle to unpkg too !